robomaker-sample-app-ci/src/aws-robomaker-sample-application-ci.ts

```diff
@@ -105,7 +101,7 @@ async function setup() {
     ];
 
     const python3Packages = [
-      "setuptools",
+      "setuptools==49.3.0",
```

Comment:
I don't think fixing the setuptools version is a good idea only because the latest setuptools version is currently broken

Reply:
looks like the second-latest version is `49.6.0`. I will update this.

Reply:
consider blacklisting 50.0.0 instead (such as setuptools != 50.0.0)

Reply:
good idea

Reply:
instead of fixing the package version, consider blacklisting 50.0.0 instead (such as ```setuptools != 50.0.0```)

Reply:
What's wrong with the latest `setuptools`?